salsa-2022-tests/tests/lru.rs

```diff
@@ -178,6 +183,9 @@ fn lru_keeps_dependency_info() {
         assert_eq!(x as usize, i);
     }
 
+    db.salsa_runtime_mut()
+        .synthetic_write(salsa::Durability::HIGH);
```

Comment:
> Any idea what's up with that? :)

@nikomatsakis (add some details, hope it's clear) `get_hot_potato(0)` was evicted due to lru. `synthetic_write` will add `current_revision`,  `get_hot_potato2(0)` and `get_hot_potato(0)` will be re-executed, since we don't keep lru dependency info in this branch. #371 ported lru improments that will make this test pass

Reply:
I see